tests/e2e/specs/activate-and-setup/setup-wizard.test.js

```diff
@@ -228,12 +228,11 @@ describe( 'Setup Wizard', () => {
 		} );
 
 		it( 'links to course creation', async () => {
-			await expect( page ).toClick( 'a', {
-				text: 'Create a course',
-			} );
```

Comment:
Updated this test to not actually open the new course page, because the editor there was causing an error in the E2E browser on 5.2 - which should be unrelated to anything we do.